### PR TITLE
Update the initial demo setup

### PR DESCRIPTION
**Pull request by kizza**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -1,2 +1,10 @@
-export * from 'react-project/webpack'
+import webpack from 'webpack'
+import { ServerConfig, ClientConfig } from 'react-project/webpack'
 
+ClientConfig.plugins.push(
+  new webpack.DefinePlugin({
+    'process.env.ACCESS_TOKEN': JSON.stringify(process.env.ACCESS_TOKEN)
```
Comment (bradparker) [inline webpack.config.js]: What's the purpose of stringifying here?

Comment (bradparker) [inline webpack.config.js]: Also `'path.through.object' === MAGIC` :D

Comment (kizza) [inline webpack.config.js]: @brad - yeah for some reason it didn't work just as is.  Regardless have used a more appropriate web plugin that does away with it :)

File: webpack.config.js [hunk 2]
```diff
@@ -1,2 +1,10 @@
-export * from 'react-project/webpack'
+import webpack from 'webpack'
+import { ServerConfig, ClientConfig } from 'react-project/webpack'
 
+ClientConfig.plugins.push(
+  new webpack.EnvironmentPlugin([
```
Comment (bradparker) [inline webpack.config.js:5]: Huh, nice

**Comment by brad** [comment]
@kizza FYI, I'm @brad :smile:

**Pull request merged**